src/Cli/dotnet/Installer/Windows/Security/NativeMethods.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.DotNet.Installer.Windows.Security
+{
+    internal class NativeMethods
+    {
+        /// <summary>
+        /// The GUID action ID for using the AuthentiCode policy provider (see softpub.h).
```

Comment:
I assume this was copied from somewhere- I don't see a softpub.h. Can we update the comment?

Reply:
yeah, that comes from the Windows SDK and yes, I copied it from my code in the SignCheck tool